bower.json

```diff
@@ -23,6 +23,6 @@
         "tests"
     ],
     "dependencies": {
-        "polymer": "Polymer/polymer"
+        "polymer": "Polymer/polymer#~1.9"
```

Comment:
Polymer recommends such selector for hybrid components: `"polymer": "Polymer/polymer#1.9 - 2"`

See: https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements